docs/atomic-install.1.md

```diff
@@ -51,6 +53,12 @@ If --display is not specified the install command will execute.
    Use this name for creating installed content for the container.
 NAME will default to the IMAGENAME if it is not specified.
 
+**--set=NAME=VALUE**
+  Set a value that is going to be used by a system container for its configuration and can be specified multiple times.  It is used only by --system.
+
+**--system**
+  Install a system container.  System containers should be used for services that are required early by the system or that cannot run in a normal container.
+
```

Comment:
thanks, I will change it to "A system container is a container that is executed out of an systemd unit file early in boot, using runc.", as at the moment we are not using ocitools